src/components/products/product-list-item/product-list-item.tsx

```diff
@@ -0,0 +1,20 @@
+import { Link } from "react-router-dom";
+import './product-list-item.scss'
+import { Product } from "../../../interfaces/products/product.interface";
+
+interface ProductProps {
+  product: Product;
+}
+
+export default function ProductListItem({product}: ProductProps) {
+  return (
+    <>
+      <tr>
+            <td>{product.category.name}</td>
+            <td>{product.name}</td>
+            <td>{product.price}</td>
+            <td className= "detail-link-col"><Link to={`/products/${product.id}`}><button className="detail-page-btn">{">"}</button></Link></td>
```

Comment:
I think some formatting is missing here because the line is too long and it should look different. Something like this:
````
<td className="detail-link-col">
  <Link to={`/products/${product.id}`}>
    <button className="detail-page-btn">
      {">"}
    </button>
  </Link>
</td>
````